docs/disaster-recovery.md

```diff
@@ -0,0 +1,442 @@
+# Failover and Failback In DR
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/)
+ is an asynchronous replication of RBD images between multiple Ceph clusters.
+This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded
+ to the associated journal before modifying the actual image.
+ The remote cluster will read from this associated journal and
+ replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or
+ manually created RBD image mirror-snapshots to replicate
+ crash-consistent RBD images between clusters.
+
+Here, Snapshot based mirroring approach is used to asynchronously
+replicate RBD images for Disaster Recovery.
+
+## Create Ceph Pools
+
+In this section we create specific ceph pools that are RBD mirroring
+ enabled for use with the DR use case.
```

Comment:
please correct the intendation.

Reply:
Thanks for looking! But the indentation here is correct. To avoid the "line-length:" linter, the sentence is written in several lines. The space in the second line is necessary as there is a space between the words "mirroring" and "enabled". Also, space cannot be left in the first line after the word "mirroring" as it would count as a trailing white space.

Reply:
oh.ok :+1: